quickwit/quickwit-control-plane/src/indexing_scheduler/scheduling/mod.rs

```diff
@@ -124,7 +125,21 @@ fn convert_physical_plan_to_solution(
                     source_id: indexing_task.source_id.clone(),
                 };
                 if let Some(source_ord) = id_to_ord_map.source_ord(&source_uid) {
-                    indexer_assignment.add_shards(source_ord, indexing_task.shard_ids.len() as u32);
+                    let source_type = &source_ord_map.get(&source_ord).unwrap().source_type;
```

Comment:
I'm always afraid of those unwraps, I'm unsure if a comment is worth it on the function or maybe replace unwrap by expect("")